search/search-client-connect/src/main/java/io/camunda/search/connect/jackson/CustomOffsetDateTimeDeserializer.java

```diff
@@ -19,9 +19,12 @@
 
 public final class CustomOffsetDateTimeDeserializer extends JsonDeserializer<OffsetDateTime> {
 
-  private DateTimeFormatter formatter;
+  private static final DateTimeFormatter OFFSET_DATE_TIME_FORMATTER_NO_COLON =
+      DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSSXXXX");
```

Comment:
@tmetzke is this formatter equal to `DateTimeFormatter.ISO_OFFSET_DATE_TIME` ?

Reply:
No, the closest equivalent to that formatter would be using `XXXXX` (5x), which can parse colon-based offsets with up to seconds-detail precision. But even that is not close enough, as the `ISO_OFFSET_DATE_TIME` also allows milliseconds and seconds to be left out in the date String.

Reply:
The `ISO_OFFSET_DATE_TIME` is actually used in the `OffsetDateTime.parse(String)` method 🙂